api/build.gradle

```diff
@@ -36,9 +39,9 @@ buildscript {
         OPENTELEMETRY_JAVA_VERSION = '2.8.0'
         OPENTELEMETRY_GOOGLE_VERSION = '0.34.0'
         SPRINGFOX_VERSION = '3.0.0'
-        SPRING_BOOT_VERSION = '3.4.3'
```

Comment:
due to inconsistencies in package resolution, we were not in fact using this version of Spring Boot.  We were actually using a 3.2 version (TODO details)

below I have downgraded our specification to the 3.2.x series, but I chose the most recent patchlevel 3.2.12, which is still pretty recent (Nov 2024)